api/api.proto

```diff
@@ -3155,11 +3155,13 @@ message SdkStorageRebalanceRequest {
   repeated  StorageRebalanceTriggerThreshold trigger_thresholds = 1;
   // TrialRun if true the job only produces steps that would be taken without making any changes
   bool trial_run = 2;
-  // PoolSelectorLabels allows selecting pools to which trigger thresholds will apply
-  map<string, string> pool_selector_labels = 3;
+  // SourcePoolSelectorLabels allows selecting pools to which trigger thresholds will apply as source
+  map<string, string> source_pool_selector_labels = 3;
```

Comment:
Instead of map<string, string> let's use the existing `LabelSelectorRequirement` type already in this proto that we use for volume placement strategies.

That will enable the API to support more primitives like NotEquals, In, Gt .. ? With a map, we are stuck with a strict equals label match.

Reply:
updated, though cli support is not available, we can special case.